src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const lines = table.getElementsByTagName('tr');
+const tableBody = table.querySelector('tbody');
+const counter = lines[0].getElementsByTagName('td');
```

Comment:
The `counter` variable is statically set to the number of columns in the first row at the time of script execution. This will not update dynamically as columns are added or removed. Consider calculating the number of columns dynamically within the event listeners to ensure accurate behavior.